frontend/app/src/components/AppView/AppView.tsx

```diff
@@ -254,7 +254,6 @@ function AppView(props: AppViewProps): ReactElement {
       {!showSidebar && appLogo && (
         <StyledSidebarOpenContainer
           chevronDownshift={sidebarChevronDownshift}
-          isCollapsed={true}
```

Comment:
Happy to talk in person, but can you explain how removing this works. Is this a react issue? Cause I see that isCollapsed will always be true and should always render, so I am unclear on how this handles the sidebar state transition